Documentation/gettingstarted/ipvlan.rst

```diff
@@ -73,13 +73,7 @@ connect to kube-apiserver.
 
 Masquerading with iptables in L3-only mode is not possible since netfilter
 hooks are bypassed in the kernel in this mode, hence L3S (symmetric) had
-to be introduced in the kernel at the cost of performance. However, Cilium
-supports its own BPF-based masquerading which does not rely in any way on
-iptables masquerading. If the ``global.installIptablesRules`` parameter is set
-to ``"false"`` and ``global.masquerade`` set to ``"true"``, then Cilium will
-use the more efficient BPF-based masquerading where ipvlan can remain in
-L3 mode as well (instead of L3S). A Linux kernel v4.16 or higher would be
-required for BPF-based masquerading.
+to be introduced in the kernel at the cost of performance.
```

Comment:
We might consider adding this BPF masquerading feature into the list of unsupported features with IPVLAN at the top of the doc:

https://docs.cilium.io/en/v1.7/gettingstarted/ipvlan/

Reply:
Good idea, adding.